app/about_us/page.js

```diff
@@ -22,7 +24,17 @@ const OurCrew = () => {
   // Use the pictures from /public/crew
   // Some inspiration ideas found in /data/inspiration_about_us
   return (
-    <p> ADD OUR CREW HERE </p>
+   <div>
+    <p>Meet the best space travelers and scientist in the world.</p>
+    <img src="/crew/image-anousheh-ansari.png" alt="Anousheh Ansari" />
+    <p>Anousheh Ansari</p>
+    <img src="/crew/image-douglas-hurley.png" alt="Douglas Hurley" />
+    <p>Douglas Hurley</p>
+    <img src="/crew/image-victor-glover.png" alt="Victor Glover" />
```

Comment:
This `alt="..."` text is not providing any extra info for the non-sighted users, so you can either change it or remove it.

- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA